stroeer/core/v1/core_article_service.proto

```diff
@@ -10,10 +10,42 @@ option java_package = "de.stroeer.core.v1";
 
 // Service to manage core articles
 service ArticleService {
-  // Returns a core article by it's id
+  // Returns a core article by it's `id`
   rpc GetArticle (GetArticleRequest) returns (stroeer.core.v1.Article) {}
+
+  // Returns pages of articles by their `home_section`
+  rpc ListArticles (ListArticlesRequest) returns (ListArticlesResponse) {}
+  // also as continuous stream
+  rpc LotsOfArticles (ListArticlesRequest) returns (stream ListArticlesResponse) {}
```

Comment:
I did not find anything about this in the protov3 spec. It would be interesting what is generated for node

Reply:
I'd like to have this as an experimental feature. Maybe we remove this later if its no use or too complicated to implement.
But the idea is appealing to make one request for `/section/` and then a continuous stream of items is delivered.

Reply:
we could also experiement locally and release it when useful